docs/UserGuide.md

```diff
@@ -235,7 +239,7 @@ Description:
 Options:
 * `u/new AMOUNT`
 
-    The `new` keyword can only be used on a property without an existing status. AMOUNT is the amount of money that is offered in the Option to Purchase
+    The `new` keyword can only be used on a property without an existing status. `AMOUNT` is the amount of money that is offered in the Option to Purchase
```

Comment:
not sure if its cut off or the significance of `new` keyword is unclear

Reply:
The `new` keyword is explaining the `new` in `u/new AMOUNT`. If it is still unclear, then maybe @dvdweien should fix it in his next PR.